ground/src/main/java/com/google/android/ground/ui/datacollection/tasks/multiplechoice/MultipleChoiceTaskViewModel.kt

```diff
@@ -49,7 +49,7 @@ class MultipleChoiceTaskViewModel @Inject constructor() : AbstractTaskViewModel(
           ?.let {
             val selected =
               if (task.isRequired) {
-                otherText != ""
+                otherText.trim() != ""
```

Comment:
You could use `!task.isRequired || otherText.isNotBlank()` here, though I don't quite get why this would mean the item is selected?